pkgs/data/fonts/cozette/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchurl, mkfontscale }:
+
+stdenv.mkDerivation rec {
+  pname = "Cozette";
+  version = "1.5.1";
+  srcs = [
+    (fetchurl {
+      url = "https://github.com/slavfox/Cozette/releases/download/v." + version
+        + "/cozette.otb";
+      sha256 = "05k45n7jar11gnng2awpmc7zk9jdlzd6wz87xx49cp75jm4z9xm8";
+    })
+
+    (fetchurl {
+      url = "https://github.com/slavfox/Cozette/releases/download/v." + version
+        + "/CozetteVector.otf";
+      sha256 = "1sqhnjpizn1wi26lc7z2zml7yr7zkcpa72mh1drvd74rlcs1ip30";
+    })
+
+    (fetchurl {
+      url = "https://github.com/slavfox/Cozette/releases/download/v." + version
+        + "/CozetteVector.ttf";
+      sha256 = "1q4ml8shv9lmyc6bwhffwvbvl92s73j7xkb0rkqvci4f0zbz7mcy";
+    })
+  ];
+
+  nativeBuildInputs = [ mkfontscale ];
+
+  sourceRoot = "./";
+
+  unpackCmd = ''
+    otName=$(basename $(stripHash $curSrc))
+    cp $curSrc ./$otName
+  '';
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/misc
+    mkdir -p $out/share/fonts/truetype
+    cp -a *.otf $out/share/fonts/misc/
+    cp -a *.otb $out/share/fonts/misc/
+    cp -a *.ttf $out/share/fonts/truetype/
+    mkfontdir "$out/share/fonts/misc"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A bitmap programming font optimized for coziness.";
+    homepage = "https://github.com/slavfox/cozette";
+    license = licenses.mit;
+    platforms = platforms.all;
+    maintainers = with maintainers; [ ];
```

Comment:
This (`maintainers`) should be set. See https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes, the last bullet. If you add yourself as the maintainer, be sure to do so in a separate commit, please.

Reply:
Done :)

Reply:
Just a slight nitpick: commit title for adding a maintainer should start with `maintainers:`, since that's the file being modified, not the module you're becoming the maintainer for ;)